.github/workflows/codeql.yml

```diff
@@ -2,8 +2,16 @@ name: "CodeQL"
 
 on:
   push:
+    paths-ignore:
+      - '**/*.md'
+      - 'dev/**'
+      - 'helm/**'
```

Comment:
```suggestion
      - '**/*.md'
      - 'dev/**'
      - 'docs/**'
      - 'helm/**'
```

Reply:
@fectrain, ah, yeah, I didn't see the `broken-link-check.py` script. The only things that won't be covered is assets like `*.png` and `*.json`, but yeah, we can revisit that later.

Reply:
@317brian  are we using the `docs/_bin/broken-link-check.py`? if not, would be nice if we could add the CodeQL exclusion.

Reply:
Also, I noticed @vtlim added a script wired into CI that checks for broken anchor links: https://github.com/apache/druid/pull/15624 

Should `broken-link-check.py` be deprecated in favor of the new script since the latter seems to have more than parity?

Reply:
@abhishekrb19 , lets do.
cc: @317brian

Reply:
We can deprecate it. I'm not sure when this script was last used, but in any case it's likely out of date. Now Docusaurus 2 and `link-lint.js` together check the links.